core/modules/ip_frag.cc

```diff
@@ -39,6 +39,12 @@ IPFrag::FragmentPkt(Context *ctx, bess::Packet *p)
 		unsigned char *orig_ip_payload;
 		uint16_t orig_data_offset;
 
```

Comment:
Comment on top of the function seems to describe reassembly. Is that right?

Reply:
Done